src/parser.ts

```diff
@@ -330,6 +332,7 @@ export class GCodeParser {
             } else if (cmd[0] === "G92") {
                 // set state
                 lastLastPoint.copy(lastPoint)
+                // TODO fix:  'parseValue' value may be zero which is also not truthy
```

Comment:
Ops, one more

Reply:
Nice find, 
should be fixed in https://github.com/aligator/gcode-viewer/pull/11
Could you please review that?